functions.php

```diff
@@ -66,15 +71,19 @@ function judge_familiar_page_navigation_box( $page ) {
 
 		$prev = get_post_meta( $page->ID, '_previous_page', true );
 		$next = get_post_meta( $page->ID, '_next_page', true );
+		$prev_title = get_post_meta( $page->ID, '_previous_page_title', true );
+		$next_title = get_post_meta( $page->ID, '_next_page_title', true );
 ?>
 		<div class="page_navigation_input">
 			<label for="page_navigation_input_prev">Previous Page</label>
-			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="text" value="<?= $prev ?>" />
+			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="number" value="<?= $prev ?>" placeholder="Page ID"/><!--
+			--><input id="page_navigation_input_prev_title" name="page_navigation_input_prev_title" type="text" value="<?= $prev_title ?>" placeholder="Page Title" />
```

Comment:
Hehe, I'd more in favor of floats than of comments affecting the layout.
But if you feel strongly about this I'll merge as is tonight.

On Mon, Apr 4, 2016, 8:44 AM Steffen <notifications@github.com> wrote:

> In functions.php
> <https://github.com/magicjudges/judge-familiar/pull/1#discussion_r58397899>
> :
>
> >  ?>
> >  		<div class="page_navigation_input">
> >  			<label for="page_navigation_input_prev">Previous Page</label>
> > -			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="text" value="<?= $prev ?>" />
> > +			<input id="page_navigation_input_prev" name="page_navigation_input_prev" type="number" value="<?= $prev ?>" placeholder="Page ID"/><!--
> > +			--><input id="page_navigation_input_prev_title" name="page_navigation_input_prev_title" type="text" value="<?= $prev_title ?>" placeholder="Page Title" />
>
> The comment is only there to make it more readable. It would actually be a
> single line and removing a blank space that is inserted to increase code
> readability by adding extra styles to compensate for it, is actually not
> the right way to do it. ;) Width adjustments by X% always rely on specific
> viewport widths / font sizes / browser rendering, so doing it this way
> really is the most clean, unless you use floats.
>
> —
> You are receiving this because you commented.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/magicjudges/judge-familiar/pull/1/files/4adf002f3c916ab853b9bae7a462be756ccc815c#r58397899>
>